utils/src/main/scala/it/pagopa/interop/commons/utils/service/impl/OffsetDateTimeSupplierImpl.scala

```diff
@@ -5,5 +5,5 @@ import it.pagopa.interop.commons.utils.service.OffsetDateTimeSupplier
 import java.time.OffsetDateTime
 
 case object OffsetDateTimeSupplierImpl extends OffsetDateTimeSupplier {
-  override def get: OffsetDateTime = OffsetDateTime.now()
+  override def get(): OffsetDateTime = OffsetDateTime.now()
```

Comment:
Taking advantage of your PR, could you please add this? 🙏 
```suggestion
  override def get(): OffsetDateTime = OffsetDateTime.now(ZoneOffset.UTC)
```
This helps avoiding unwanted and unexpected behaviours

Reply:
How about 2 method, one that accepts a zone offset and the other one that uses UTC as the default value calling the implementation? The definition could be in the trait itself. I'll write down something...

Reply:
I'd rather have only one method to avoid using the wrong one.
IMO all our time references should be in UTC

Reply:
This approach won't prevent implementing the function in other ways, while my approach forces the implementation to at least use the argument. I don't have strong opinions about it; let's choose :)

Reply:
true but, at least if I'm not missing something, right now it would be an error having anywhere a timezone different from UTC.
Not giving the choice (until strictly required) would prevent it.

Reply:
> right now, it would be an error having anywhere a timezone different from UTC.
Because we say it, not because we enforced it somehow, unluckily.

Reply:
correct, that's why we should always use the date time supplier.
And by configuring the timezone in this implementation we would enforce the expected behaviour.

Reply:
I just sent it btw